app/proguard-rules.pro

```diff
@@ -118,8 +118,13 @@
 ## Rules for NewPipeExtractor
 -keep class org.schabi.newpipe.extractor.timeago.patterns.** { *; }
 -keep class org.mozilla.javascript.** { *; }
--keep class org.mozilla.classfile.ClassFileWriter
+-keep class org.mozilla.javascript.engine.** { *; }
+-dontwarn org.mozilla.javascript.JavaToJSONConverters
 -dontwarn org.mozilla.javascript.tools.**
+-keep class javax.script.** { *; }
+-dontwarn javax.script.**
+-keep class jdk.dynalink.** { *; }
+-dontwarn jdk.dynalink.**
```

Comment:
What's the source of these additional Proguard rules?

I assume they have become necessary with commits **after** the 0.24.5 release?

Reply:
They're from [NewPipe](https://github.com/TeamNewPipe/NewPipe/pull/12104). Without them, the build will fail due to missing classes.